src/community/jms-cluster/jms-commons/src/main/java/org/geoserver/cluster/configuration/JMSConfiguration.java

```diff
@@ -168,26 +170,22 @@ public final boolean override(String nameKey, Object defaultVal) {
     }
 
     public void loadConfig() throws IOException {
-        final File config = new File(configPathDir, CONFIG_FILE_NAME);
-        FileInputStream fis = null;
+        final Resource config = configPathDir.get(CONFIG_FILE_NAME);
```

Comment:
I guess this was done because some people like to declare anything that /can/ be final as final (rather than everything that /must/ be final, for use in an inner class). In this case the variable will definitely not need to be changed, and that could be considered sufficient reason.  See http://www.javapractices.com/topic/TopicAction.do?Id=23 and http://programmers.stackexchange.com/questions/115690/why-declare-final-variables-inside-methods